network-protection/network-protection-impl/src/main/java/com/duckduckgo/networkprotection/impl/configuration/WgTunnel.kt

```diff
@@ -80,6 +83,7 @@ class RealWgTunnel @Inject constructor(
                 }
         } catch (e: Throwable) {
             logcat(LogPriority.ERROR) { "Error getting WgTunnelData: ${e.asLog()}" }
+            crashLogger.logCrash(Crash("vpn_register_key_error", e))
```

Comment:
Note: this might spike the crashes  due to errors caused by not internet connection (https://app.asana.com/0/0/1205732562346501/f)